packages/node/src/index.ts

```diff
@@ -65,6 +65,8 @@ const INTEGRATIONS = {
 
 export { INTEGRATIONS as Integrations, Handlers };
 
+export { sentryTrpcMiddleware } from './trpc';
```

Comment:
Not sure if lib-specific code should be exported directly from our core node package tbh. And if so, we can maybe put it under `handlers.ts`, where our express-related code lives already?

Reply:
I wouldn't know where else to put it. I think it is fine here, considering we also have stuff like specific DB integrations exported from the node pkg. Do you know a better place?

Also, I wouldn't put it into handlers. It's not really a handler lol. Also, I don't think it matters too much in which file we put it. I just wanted to be able to find it quickly if I fuzzy search for "trpc".

Reply:
> I think it is fine here, considering we also have stuff like specific DB integrations exported from the node pkg.

We don't, it's exported from tracing package :P

> It's not really a handler lol.

Neither are express `request` and `tracing` "middlewares" that we call handlers, yet we expose them via `Handlers` :P

wdyt @AbhiPrasad? If we leave it here, I'd simplify the name to `tRPCMiddleware` as import `'sentrySomething' from 'sentry'` sounds strange.

Reply:
> We don't, it's exported from tracing package :P

This will change, we want to `rm -rf` the tracing package. https://github.com/getsentry/sentry-javascript/issues/7346

> I'd simplify the name to tRPCMiddleware as import 'sentrySomething' from 'sentry' sounds strange.

Sounds fine to me. Only concern we might have is that it might collide with other packages, which might be annoying for vscode autocomplete (not a huge deal, but still some DX we need to think about).